cmd/feed-ingress/main.go

```diff
@@ -67,7 +69,7 @@ func main() {
 		os.Exit(-1)
 	}
 
-	os.Exit(0)
+	select {}
```

Comment:
Blocks forever, since I made `controller.Start()` non blocking now (so we can handle shutdown in the signal handler).